src/pieces/commands/search_command.py

```diff
@@ -0,0 +1,60 @@
+from pieces_os_client.api.assets_api import AssetsApi
+from pieces_os_client.api.search_api import SearchApi
+
+from pieces.settings import Settings
+from pieces.gui import show_error, print_asset_details
+from pieces.assets.assets_api import AssetsCommandsApi
+
+
+def search(query, **kwargs):
+    global asset_ids 
+
+    search_type = kwargs.get('search_type', 'assets')
+
+    # Join the list of strings into a single search phrase
+    search_phrase = ' '.join(query)
+
+    # Call the API function with the search phrase and type
+    query = search_phrase
```

Comment:
Not sure if we should do this for this pr or if we do a security overview later (since this pr is about speed) but just a mental note we should probably do some sanitation of user queries since Pieces is big on security.